addons/mail/static/src/components/composer/composer_tests.js

```diff
@@ -83,10 +83,77 @@ QUnit.test('composer text input: basic rendering', async function (assert) {
         1,
         "should have editable part inside composer text input"
     );
+    assert.strictEqual(
+        document.querySelector(`.o_ComposerTextInput_textarea`).placeholder,
+        "Send a message to followers...",
+        "should have 'Send a message to followers...' as placeholder composer text input"
+    );
+});
+
+QUnit.test('composer text input: basic rendering when logging note', async function (assert) {
+    assert.expect(5);
+
+    await this.start();
+    const composer = this.env.models['mail.composer'].create({ isLog: true });
```

Comment:
I would create a non-channel `mail.thread` instead of a `mail.composer`. It's quite dangerous to have composer without thread, and test is weird without a thread.

(previous test has the same issue)